src/components/Inputs/Input.tsx

```diff
@@ -1,24 +1,24 @@
-import cn from 'classnames';
 import React, { forwardRef } from 'react';
-import { Label } from './Label';
+import cn from 'classnames';
+import Label from './Label';
 
 interface Props extends React.ComponentPropsWithRef<'input'> {
   label?: string;
   error?: string;
   id: string;
 }
 
-export const Input = forwardRef<HTMLInputElement, Props>(({ className, id, label, type, error, ...rest }, ref) => {
-  return (
-    <div className={cn('flex w-full flex-col gap-1', className)}>
-      {(label || error) && (
-        <Label forId={id} error={error}>
-          {label}
-        </Label>
-      )}
-      <input type={type ?? 'text'} className="border-b-2 text-[#333F4C]" id={id} ref={ref} {...rest}></input>
-    </div>
-  );
-});
+const Input = forwardRef<HTMLInputElement, Props>(({ className, id, label, type, error, ...rest }, ref) => (
```

Comment:
Don't believe there is a way to change forwardRef arrow function to normal function. Do you know if you can? @mbaltrimas

Reply:
would something like this work? 
```
function Input({ className, id, label, type, error, ...rest }: Props, ref: Ref<HTMLInputElement>): ReactElement {
  return (
    <div className={cn('flex w-full flex-col gap-1 text-[16px]', className)}>
      {(label || error) && (
        <Label forId={id} error={error}>
          {label}
        </Label>
      )}
      <input
        type={type ?? 'text'}
        className="text-ak-grey-1 text-[14px] placeholder:text-[#333F4C] focus:pl-1 focus:placeholder:text-[#A2AAB6]"
        id={id}
        ref={ref}
        {...rest}
      ></input>
    </div>
  );
}

export default forwardRef(Input);
```

Reply:
Nice that worked, thank you! @mbaltrimas